### ISSUE TITLE
issue of escape character 

### ISSUE THREAD

**Issue by martingg88** [orientechnologies/orientdb#5243]
i try to save the following json value into class and the orientdb will try to add extra escape character to the json value and it look weird. 

you will notice there are extra slashes around "welcome" word as follows: 
 ```
 input = {"content":"alert(\"welcome\")"}

 output = {\"content\":\"alert(\\\"welcome\\\")\"}
```

**Comment by kiejo** [comment]
I think this is related to #1787

**Comment by luigidellaquila** [comment]
Hi @martingg88 

This issue is fixed in both 2.1.17 and 2.2.0 

I'm closing it, please reopen if you experience any similar problems

Thanks

Luigi
